service-front/module/Application/src/Model/Service/Analytics/GoogleAnalyticsService.php

```diff
@@ -4,13 +4,10 @@
 
 use Application\Model\Service\AbstractService;
 use Exception;
-use Opg\Lpa\Logger\LoggerTrait;
```

Comment:
just checking if this removal, non replacement was intentional.

Reply:
Good catch @williamfalconeruk this does need to reference the new logger class.